addons/traefik/disable

```diff
@@ -1,16 +1,11 @@
-#!/usr/bin/env bash
+#!/bin/bash -e
 set -e
 
-source $SNAP/actions/common/utils.sh
-CURRENT_DIR=$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)
+HELM="$SNAP_DATA/bin/helm3 --kubeconfig=$SNAP_DATA/credentials/client.config"
```

Comment:
I think this also needs a `microk8s enable helm3` before everything else.

Reply:
Done